modules/jdbc-test/src/main/java/org/testcontainers/jdbc/AbstractJDBCDriverTest.java

```diff
@@ -117,7 +117,9 @@ private void performTestForJDBCParamUsage(HikariDataSource dataSource) throws SQ
         String databaseQuery = "SELECT DATABASE()";
         // Postgres does not have Database() as a function
         String databaseType = ConnectionUrl.newInstance(jdbcUrl).getDatabaseType();
-        if (databaseType.equalsIgnoreCase("postgresql") || databaseType.equalsIgnoreCase("postgis")) {
+        if (databaseType.equalsIgnoreCase("postgresql") ||
+            databaseType.equalsIgnoreCase("postgis") ||
+            databaseType.equalsIgnoreCase("timescaledb")) {
             databaseQuery = "SELECT CURRENT_DATABASE()";
         }
```

Comment:
This is pretty ugly and should be made more generic, but that would be a different PR.